public/theme/boltcms-2021/showcases.twig

```diff
@@ -0,0 +1,47 @@
+{% extends 'partials/_master.twig' %}
+
+{% block main %}
+    <body class="loading showcase">
+    <main>
+        <div data-scroll>
+            <div class="subheader">
+                <h1>{{ contenttype.name }}</h1>
+                <p>A fine selection of websites made with Bolt</p>
+            </div>
+            <div class="content">
+                {% for showcase in showcases|slice(0, 9) %}
+                    <div class="item">
+                        {% if showcase.screenshot is not empty %}
+                            <div class="item__img-wrap"
+                                 style="--image: url({{ showcase.screenshot|thumbnail(700, 400) }});">
+                                <div class="item__img item__img--t{{ random(1, 3) }}"></div>
+                            </div>
+                        {% endif %}
+                        <div class="item__caption">
+                            <h2 class="item__caption-title">{{ showcase.title }}</h2>
+                            <p class="item__caption-url"><a href="{{ showcase.website_link }}" target="_blank"
+                                                            rel="noopener noreferrer">{{ showcase.website_link }}</a>
+                            </p>
+                            <p class="item__caption-copy">{{ showcase.website_name }}</p>
+                            <p class="item__caption-url">Build by: <span
+                                        style="color: var(--color-primary)">{{ showcase.contributor }}</span>
```

Comment:
Let's put the `color: var(--color-primary)` in the css class where it belongs, as @Dsduit suggested!

Also, as discussed this morning, let's make the `Built by {{ showcase.contributor }}` optional (so, only if it is _not_ empty, we include it).